share/github-backup-utils/ghe-restore-pages

```diff
@@ -23,7 +23,17 @@ GHE_HOSTNAME="$1"
 : ${GHE_RESTORE_SNAPSHOT:=current}
 
 # Find the pages to restore
-pages_paths=$(cd $GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/ && find pages -mindepth 5 -maxdepth 5 | cut -d / -f2-)
+
+check_pages=$(cd $GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/ && find pages -mindepth 5 -maxdepth 5 2> /dev/null)
+
+# Check if there were pages in backup, if no skip restore 
```

Comment:
What's the functional difference in splitting this out from the next check? They both exit early it looks like -- just a more pertinent echo message? Do we ever expect this one to trigger but not the next one?

Reply:
Yeah, so logically they are a little different in scenarios. Our change is when the backup skipped having pages altogether (there is no pages dir which caused an ugly error), but the change from before was there were pages in the backup, but it was empty. Maybe it's not that different for a customer, but I thought of them as two scenarios, hence different wording made sense